packages/ui-axe-check/src/runAxeCheck.ts

```diff
@@ -21,30 +21,39 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
 import axeCore from 'axe-core'
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'element' implicitly has an 'any' type.
-export default async function runAxe(element, options = {}) {
-  let result = true
-  const context = {
+type AxeCheckOptions = {
+  exclude?: string | string[]
+  /**
+   * Violations to ignore/filter out. 'color-contrast' is automatically excluded
+   * because we test color contrast in theme tests.
+   */
+  ignores?: string[]
+}
+
+export default async function runAxe(
+  element: string,
```

Comment:
From the build error it seems that we pass HTMLElements to the `runAxeCheck(element, options)` in the ui-test-queries `accessible` util.

Reply:
its a bug in axe-check's typing I've opened a bug and did a cast as a workaround

Reply:
oh cool, thanks!